pandas/tests/test_strings.py

```diff
@@ -1008,6 +1008,13 @@ def test_replace(self):
                     values = klass(data)
                     pytest.raises(TypeError, values.str.replace, 'a', repl)
 
+    def test_replace_single_pattern(self):
+        values = Series(['abc','123'])
+
+        result = values.str.replace('.', 'foo')
+        exp = Series(['foofoofoo', 'foofoofoo'])
```

Comment:
Can you name this `expected`?

Reply:
Sure, should I make another PR?

Reply:
No just add as a commit and push on the same branch. 

Looks like CI failed too -haven’t checked but make sure tests pass locally before pushing

Reply:
Yes, i've seem that.
Actually, i know what is the problem.
The problem is the `test test_pipe_failures`. It was built to test a char replacement: pipe to white space. 
But, pipe is a regex code too. 
When i fixed the replace behavior, this test was broken.
My proposal is change this test to pass the regex=False parameter. Like below:

```
    def test_pipe_failures(self):
        # #2119
        s = Series(['A|B|C'])

        result = s.str.split('|')
        exp = Series([['A', 'B', 'C']])

        tm.assert_series_equal(result, exp)

        result = s.str.replace('|', ' ', regex=False)
        exp = Series(['A B C'])

        tm.assert_series_equal(result, exp)
```

Reply:
What to think about that?